prow/deck/jobs/jobs.go

```diff
@@ -282,7 +282,12 @@ type byPJStartTime []prowapi.ProwJob
 func (a byPJStartTime) Len() int      { return len(a) }
 func (a byPJStartTime) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
 func (a byPJStartTime) Less(i, j int) bool {
-	return a[i].Status.StartTime.Time.After(a[j].Status.StartTime.Time)
+	if a[i].Status.StartTime.Time != a[j].Status.StartTime.Time {
+		return a[i].Status.StartTime.Time.After(a[j].Status.StartTime.Time)
+	}
+	// Start time only has second granularity and we often start many jobs in the
+	// same second. Use the name as tie breaker.
+	return a[i].Spec.Job < a[j].Spec.Job
```

Comment:
Can't we have nil pointer here in `Spec` because the job is not agent=kubernetes?
I don't think that would affect instances in https://github.com/kubernetes/test-infra/issues/25998 but this seems like a bug.

Reply:
@BenTheElder no, these are prowjob CRs. Both `Spec` and `Job` are value fields that can not be nil: https://github.com/kubernetes/test-infra/blob/99638a6491186bb1410faf50b7ceaff4ea74b376/prow/apis/prowjobs/v1/types.go#L141

Reply:
sourcegraph is confused and thinks .Spec is a pointer to podspec when I hover here 🙃 
I can see that is incorrect.

Reply:
I can load a build just fine w/ runlocal...